collection-scripts/gather_sos

```diff
@@ -80,7 +80,7 @@ gather_node_sos () {
           mkdir -p \"${TMPDIR}\" && \
           sudo podman rm --force toolbox-osp;  \
           sudo --preserve-env podman pull --authfile /var/lib/kubelet/config.json registry.redhat.io/rhel9/support-tools && \
-          toolbox sos report --batch $SOS_LIMIT --tmp-dir=\"${TMPDIR}\" && \
+          toolbox sos report --batch --all-logs $SOS_LIMIT --tmp-dir=\"${TMPDIR}\" && \
```

Comment:
i assunme this enabels the all_logs blocs in each of the plugins like this

https://github.com/rebtoor/sos/blob/a2f7bd9cec551c3283badb64c1605509a9aae7c0/sos/report/plugins/openstack_nova.py#L93-L96

for the OpenStack services those will largely be empty now as all opesntack service whousl be logging to the journal now
however there might be some cases where that is not the case so adding this i think makes sense overall

Reply:
Yes, that is correct, it enables all the `all_logs` blocks of the plugins that are enabled.

I wanted to limit the `all_logs` to the `logs` plugin gathering, but I couldn't find a way to enable it just for a single plugin.  Using `-k logs.all_logs` fails because there is no such plugin specific option. :-(